### ISSUE TITLE
Unfortunately, the playlist does not download.

### ISSUE THREAD

**Issue by yhackup** [shaked6540/YoutubePlaylistDownloader#267]
Unfortunately, the playlist does not download. The percentages are full as if it is downloading but there is no video in the download folder.

**Comment by shaked6540** [comment]
Try running it as administrator

**Comment by yhackup** [comment]
Yes, it downloads like this. You need to force the application to run as administrator
